src/utils/Utils.js

```diff
@@ -213,6 +213,25 @@ class Utils {
         return mainSort ? mainSortFunc(a, b) : 0;
       };
     }
+    if (sortOn === 'contactOutcomeDate') {
```

Comment:
I would have written it this way for simplicity and to merge the date comparison:

  if (sortOn === 'contactOutcomeDate') {
    return (a, b) => {
      const dateA = a.contactOutcome.date;
      const dateB = b.contactOutcome.date;
      if (!dateA || !dateB) {
        return dateA ? -1 * mult : 1 * mult;
      }
      if (dateA !== dateB) {
        return (dateA < dateB ? -1 : 1) * mult;
      }
      return mainSort ? mainSortFunc(a, b) : 0;
    };
  }